rovercode/chainable_rgb_leds_manager.py

```diff
@@ -0,0 +1,38 @@
+"""Class for managing the motor state."""
+
+import logging
+logging.basicConfig()
+LOGGER = logging.getLogger(__name__)
+LOGGER.setLevel(logging.getLevelName('INFO'))
+
+
+class ChainableRgbLedsManager:
+    """Object to manage RGB LEDs."""
+
+    def __init__(self, port, count, driver):
+        """Construct a RgbLedManager."""
+        self.port = port
+        self.count = count
+        self.driver = driver
+        self.driver.setup(port, self.count)
+        LOGGER.info("RGB LED manager initialized")
+
+    def set_all_led_colors(self, red, green, blue):
+        """Set all LEDs to the same color."""
+        for led in range(self.count):
+            self.set_led_color(led, red, green, blue)
+
+    def set_led_color(self, led, red, green, blue):
+        """Set LED color."""
+        if led > self.count - 1:
```

Comment:
Might be better to check `led in range(self.count)` so that it handles negative as well

Reply:
:+1: